tests/citycoin-tardis-v2.test.ts

```diff
@@ -0,0 +1,371 @@
+import { describe, assertEquals, types, Account, run, Chain, it, beforeEach} from "../deps.ts";
+import { CoreModel } from "../models/core.model.ts";
+import { TokenModel } from "../models/token.model.ts";
+import { TardisModel } from "../models/tardis.model.ts";
+import { Accounts, Context } from "../src/context.ts";
+
+let ctx: Context;
+let chain: Chain;
+let accounts: Accounts;
+let core: CoreModel;
+let token: TokenModel;
+let tardis: TardisModel;
+
+beforeEach(() => {
+  ctx = new Context();
+  chain = ctx.chain;
+  accounts = ctx.accounts;
+  core = ctx.models.get(CoreModel);
+  token = ctx.models.get(TokenModel);
+  tardis = ctx.models.get(TardisModel);
+})
+
+describe("[CityCoin Tardis]", () => {
+  describe("HISTORICAL ACTIONS", () => {
+    describe("get-historical-balance()", () => {
+      it("succeeds and returns the CityCoin balance at a prior block height", () => {
+        // arrange
+        const wallet_1 = accounts.get("wallet_1")!;
+        const mintAmount = 100;
+
+        // act
+        chain.mineEmptyBlock(100);
+        chain.mineBlock([
+          token.ftMint(mintAmount, wallet_1)
+        ]);
+        chain.mineEmptyBlock(100);
+        chain.mineBlock([
+          token.ftMint(mintAmount, wallet_1)
+        ]);
+
+        const result1 = tardis.getHistoricalBalance(1, wallet_1).result;
+        const result2 = tardis.getHistoricalBalance(101, wallet_1).result;
+        const result3 = token.getBalance(wallet_1).result;
```

Comment:
```suggestion
        // arrange
        const wallet_1 = accounts.get("wallet_1")!;
        const mintAmount = 100;

        chain.mineEmptyBlock(100);
        chain.mineBlock([
          token.ftMint(mintAmount, wallet_1)
        ]);
        chain.mineEmptyBlock(100);
        chain.mineBlock([
          token.ftMint(mintAmount, wallet_1)
        ]);

        // act
        const result1 = tardis.getHistoricalBalance(1, wallet_1).result;
        const result2 = tardis.getHistoricalBalance(101, wallet_1).result;
        const result3 = token.getBalance(wallet_1).result;
```
Minting coins is something that is part of test arrange activity.